statix.runtime/trans/statix/runtime/menus.str

```diff
@@ -24,5 +24,5 @@ rules
     with
       filename := "project.stxlib"
     ; sg := <stx--get-scopegraph;strip-annos> a*
-    ; s* := <collect-all(stx--is-scope);make-set> sg
+    ; s* := <collect-all(stx--is-scope, TupleToList; concat); imset-from-list; imset-to-list> sg
```

Comment:
`TupleToList; concat` is also available as `conc`

Reply:
Thank you; I was unaware of this.